db/migrate/20220525122759_init_schema.rb

```diff
@@ -0,0 +1,237 @@
+      t.jsonb 'properties', default: '{}', null: false
+      t.index ['billable_metric_id'], name: 'index_charges_on_billable_metric_id'
+      t.index ['plan_id'], name: 'index_charges_on_plan_id'
+    end
+
+    create_table 'coupons', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'organization_id', null: false
+      t.string 'name', null: false
+      t.string 'code'
+      t.integer 'status', default: 0, null: false
+      t.datetime 'terminated_at'
+      t.bigint 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.integer 'expiration', null: false
+      t.integer 'expiration_duration'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index %w[organization_id code], name: 'index_coupons_on_organization_id_and_code', unique: true, where: '(code IS NOT NULL)'
+      t.index ['organization_id'], name: 'index_coupons_on_organization_id'
+    end
+
+    create_table 'credits', force: :cascade do |t|
+      t.uuid 'invoice_id'
+      t.uuid 'applied_coupon_id'
+      t.bigint 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index ['applied_coupon_id'], name: 'index_credits_on_applied_coupon_id'
+      t.index ['invoice_id'], name: 'index_credits_on_invoice_id'
+    end
+
+    create_table 'customers', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.string 'customer_id', null: false
+      t.string 'name'
+      t.uuid 'organization_id', null: false
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.string 'country'
+      t.string 'address_line1'
+      t.string 'address_line2'
+      t.string 'state'
+      t.string 'zipcode'
+      t.string 'email'
+      t.string 'city'
+      t.string 'url'
+      t.string 'phone'
+      t.string 'logo_url'
+      t.string 'legal_name'
+      t.string 'legal_number'
+      t.float 'vat_rate'
+      t.index ['customer_id'], name: 'index_customers_on_customer_id'
+      t.index ['organization_id'], name: 'index_customers_on_organization_id'
+    end
+
+    create_table 'events', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'organization_id', null: false
+      t.uuid 'customer_id', null: false
+      t.string 'transaction_id', null: false
+      t.string 'code', null: false
+      t.jsonb 'properties', default: {}, null: false
+      t.datetime 'timestamp', precision: nil
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index ['customer_id'], name: 'index_events_on_customer_id'
+      t.index %w[organization_id code], name: 'index_events_on_organization_id_and_code'
+      t.index %w[organization_id transaction_id], name: 'index_events_on_organization_id_and_transaction_id', unique: true
```

Comment:
Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)